.pullapprove.yml

```diff
@@ -0,0 +1,42 @@
+version: 3
+
+groups:
+  pullapprove-admins:
+    conditions:
+    - "'.pullapprove.yml' in files"
+
+    reviews:
+      required: 1
```

Comment:
Does this mean we always need a 👍 from secops?

Reply:
Only for changes to the pullapprove file

Reply:
I should have realized that... `- "'.pullapprove.yml' in files"` that is the boundary of my knowledge. 👍